app/mailers/prison_mailer.rb

```diff
@@ -34,4 +34,18 @@ def rejected(visit)
            prisoner: visit.prisoner_full_name
          )
   end
+
+  def cancelled(visit)
+    @visit = visit
+
+    headers('X-Priority' => '1 (Highest)', 'X-MSMail-Priority' => 'High')
```

Comment:
This is a bit fiddly. Could we extract it to a method that explains the intention?